deploy/docker/build.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+IMAGE=${1-heapster:canary}
```

Comment:
That's what was in the original build.sh:

```
-docker build -t heapster:canary .
+docker build -t $IMAGE .
```

What would you call it? The Makefile uses `$PREFIX/heapster:$TAG`, but I didn't know what prefix to use here.